store/rootmulti/store.go

```diff
@@ -851,15 +893,22 @@ loop:
 			if !ok || store == nil {
 				return snapshottypes.SnapshotItem{}, sdkerrors.Wrapf(types.ErrLogic, "cannot import into non-IAVL store %q", item.Store.Name)
 			}
-			importer, err = store.Import(int64(height))
+			// Importer height must reflect the node height (which usually matches the block height, but not always)
+			rs.logger.Debug("Restore", "store", item.Store.Name, "version", item.Store.Version)
+			if item.Store.Version == 0 {
+				fmt.Printf("!!!!!!!!!!!!!!!!!  Restore Error, Snapshot Missing Version, Use Another Peer !!!!!!!!!!!!!!!!")
+				rs.logger.Error("Restore Error, Snapshot Missing Version, Use Another Peer", "store", item.Store.Name, "version", item.Store.Version)
+				return snapshottypes.SnapshotItem{}, sdkerrors.Wrap(sdkerrors.ErrLogic, "Snapshot Without Store Version Not Supported, Use Another Peer")
```

Comment:
nit: Please do not capitalize or punctuate errors.

Reply:
Suggestion noted.  This code is no longer relevant to the scope of this PR